validator/testcases/javascript/instanceactions.py

```diff
@@ -128,6 +128,11 @@ def _create_script_tag(traverser):
         description="Dynamic creation of script nodes can be unsafe if "
                     "contents are not static or are otherwise unsafe, "
                     "or if `src` is remote.",
+        signing_help="Please avoid using <script> tags to load scripts. "
+                     "For potential alternatives, please see "
+                     "https://developer.mozilla.org/en-US/Add-ons/"
```

Comment:
We have a mix, some with `/en-US/`, some without. I thought about removing it, but translations of pages like this tend to be sketchy/out-of-date, and given that the validator messages are not localized anyway, I'd prefer to link to the English version by default.